dojo/tools/scout_suite/parser.py

```diff
@@ -0,0 +1,137 @@
+        for service, items in list(last_run["summary"].items()):
+            test_description = "%s\n**%s** \n" % (test_description, service.upper())
+            test_description = "%s\n* **Checked Items:** %s\n" % (test_description, items["checked_items"])
+            test_description = "%s* **Flagged Items:** %s\n" % (test_description, items["flagged_items"])
+            test_description = "%s* **Max Level:** %s\n" % (test_description, items["max_level"])
+            test_description = "%s* **Resource Count:** %s\n" % (test_description, items["resources_count"])
+            test_description = "%s* **Rules Count:** %s\n\n" % (test_description, items["rules_count"])
+        test.description = test_description
+
+        scoutsuite_findings = []
+
+        # Configured Services
+        for service in list(data["services"].items()):
+            for service_item in service:
+                if "findings" in service_item:
+                    for name, finding in list(service_item["findings"].items()):
+                        if finding["items"]:
+                            description_text = ""
+                            for name in finding["items"]:
+                                description_text = description_text + "**Location:** " + name + "\n\n---\n"
+                                description_text = description_text + "\n"
+                                key = name.split('.')
+                                i = 1
+                                lookup = service_item
+                                while i < len(key):
+                                    if key[i] in lookup:
+                                        if (type(lookup[key[i]]) is dict):
+                                            lookup = lookup[key[i]]
+                                            if (key[i - 1] == "security_groups" or key[i - 1] == "PolicyDocument"):
+                                                break
+                                    i = i + 1
+
+                                self.recursive_print(lookup)
+                                description_text = description_text + self.item_data
+                                self.item_data = ""
+
+                            mobsf_item = {
+                                "category": "Mobile Permissions",
+                                "title": finding["description"],
+                                "severity": finding["level"],
+                                "description": description_text,
+                                "references": finding["references"]
+                            }
+                            scoutsuite_findings.append(mobsf_item)
+
+        for scoutsuite_finding in scoutsuite_findings:
+            title = strip_tags(scoutsuite_finding["title"])
+            sev = self.getCriticalityRating(scoutsuite_finding["severity"])
+            description = scoutsuite_finding["description"]
+            references = scoutsuite_finding["references"]
```

Comment:
@hasantayyar the `references` attributes is a simple Textfield. We need to convert it to a string I think.

Reply:
Right.

It is being auto converted now

![image](https://user-images.githubusercontent.com/88295/104211649-a056ae80-5434-11eb-903c-da37f4fabcb8.png)

Better to convert to string.

Reply:
Pushed the change. 
References is an array in the scan report. So I safely joined the array with whitespace